bbc/simorgh
# WSTEAM1-1159: Uploader Text Input Enforced Colours Border Fix

Resolves JIRA [WSTEAM1-1159](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1159)

Overall changes
======
Adds a border for enforced colours mode on the text inputs of the uploader form

Code changes
======

- Adds border colour using canvasText system colour for enforced colours
- Removes browser applied defaul outline for enforced colours
- Updates snapshots

Testing
======
1. Open Firefox or Edge on windows
2. Visit http://localhost:7081/somali/send/u130092370?renderer_env=test
3. Enables custom colours in firefox, or enable high contrast mode in windows
4. See text inputs using text colour as the border colour


Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
